### PR TITLE
feat(banner): prefix change

### PR DESCRIPTION
**Pull request by alexandereneroth**
**Describe pull-request**  

- Updated the prefix for the componet.

**Solving issue**  
Fixes: https://tegel.atlassian.net/browse/DTS-1625

**How to test**  
1. Go to storybook
2. Check in component
3. Check the styling and the API.

### CODE REVIEW

File: core/src/components/banner/tds-banner.scss [hunk 1]
```diff
@@ -74,14 +74,14 @@
...
     button {
       padding-right: 16px;
       padding-top: 14px;
       background-color: transparent;
       border: none;
-      color: var(--sdds-banner-x-color);
+      color: var(--tds-banner-x-color);
 
       &:hover {
         cursor: pointer;
```
Comment (theJohnnyMe) [inline core/src/components/banner/tds-banner.scss:87]: I could not get there, but please check line 94 - Maybe mixing could be used or create a local variable for focus for this component. The banner should not rely on focus variables from other components.

File: core/src/components/banner/tds-banner.tsx
Comment (mJarsater) [inline core/src/components/banner/tds-banner.tsx:1]: Filename should be banner.tsx

File: core/src/components/banner/tds-banner.stories.tsx
Comment (mJarsater) [inline core/src/components/banner/tds-banner.stories.tsx:1]: Filename should be banner.stories.tsx

File: core/src/components/banner/tds-banner.scss
Comment (mJarsater) [inline core/src/components/banner/tds-banner.scss:1]: Filename should be banner.scss

**Review by mJarsater — CHANGES_REQUESTED** [review]
Some filenames that needs updates

File: core/src/components/banner/tds-banner.scss (hunk 1 above)
Comment (alexandereneroth) [reply core/src/components/banner/tds-banner.scss:88]: should be fixed now

File: core/src/mixins/_focus-state.scss [hunk 2]
```diff
@@ -1,4 +1,4 @@
 @mixin tds-focus-state {
-  outline: 2px solid var(--sdds-blue-400);
+  outline: 2px solid var(--tds-blue-400);
```
Comment (mJarsater) [inline core/src/mixins/_focus-state.scss:2]: Nice 🧐

**Review by mJarsater — APPROVED** [review]
Loooking goood 🦄

**Review by theJohnnyMe — APPROVED** [review]
Great work!

**Pull request merged**